server/accounts.go

```diff
@@ -1717,14 +1717,15 @@ func (a *Account) checkForReverseEntries(reply string, checkInterest, recursed b
 	var _rs [64]string
```

Comment:
It will likely now make sense to just do `rs := make([]string, 0, len(a.imports.rrMap))` to prevent reallocations.

Reply:
Fixed, please take a quick look.